src/subscription_manager/cli_command/abstract_syspurpose.py

```diff
@@ -503,7 +503,7 @@ def _check_result(self, expectation, success_msg, command, attr):
         if result and not expectation(result):
             advice = SP_ADVICE.format(command=command)
             value = result[attr]
-            msg = _(SP_CONFLICT_MESSAGE.format(attr=attr, download_value=value, advice=advice))
+            msg = _(SP_CONFLICT_MESSAGE).format(attr=attr, download_value=value, advice=advice)
```

Comment:
you can remove `_()` altogether here, as `SP_CONFLICT_MESSAGE` is already marked for translation